### ISSUE TITLE
change learning rate in `.fit()` call?

### ISSUE THREAD

**Issue by jamesafranke** [pycroscopy/atomai#60]
Amazing package! Thanks so much for the development! 

Is it possible to change learning rate within the `.fit` call on a atomai model? For example:

```
from atomai.models import rVAE

rvae = rVAE( input_dim, latent_dim = 2, conv_encoder = True, numlayers_encoder = 3, numlayers_decoder = 3 )
rvae.fit( training_stack, training_cycles = 100, batch_size = 100, **lr = 0.0005** )
```

I am unsure how to pass the correct params to `torch.optim.Adam()` in  `.compile_trainer()`. Should it look something like?:

```
from atomai.models import rVAE
import torch

rvae = rVAE( input_dim, latent_dim = 2, conv_encoder = True, numlayers_encoder = 3, numlayers_decoder = 3 )
rvae.compile_trainer( imstack_train, optimizer = torch.optim.Adam( params = ??, lr = 0.0005 ), training_cycles = 100, batch_size = 100)
rvae.fit()
```

Thanks!

**Comment by ziatdinovmax** [comment]
@jamesafranke Thanks!
You are correct that one has to pass a torch optimizer to change the default learning rate and other learning parameters. I felt that this was a better option than introducing too many additional keyword arguments. You can do it the following way:
```python3
optimizer = lambda p: torch.optim.Adam(p, lr=1e-3)  # specify your learning rate
vae.fit(X_train, optimizer=optimizer, training_cycles=200)
```
Hope this helps.

**Comment by jamesafranke** [comment]
works like a charm. Thanks @ziatdinovmax !!
